### PR TITLE
ci: Update as per Q3 audit findings

### PR DESCRIPTION
**Pull request by mishomihov00**
**Description**:

Update as per Q3 audit findings

**Related issue(s)**:

Related to #8832 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
This needs to be discussed. This is revoking ownership of these files from mirror node team and transferring them to another team. I thought with hiero we were moving away from this approach.

**Comment by mishomihov00** [comment]
@steven-sheehy The CODEOWNERS file is primarily used for assigning automatic reviewers for PR's. Of course, I can add the "mirror-node" team if you want, but if I don't, I don't think that this will change anything major. After all this file was missing before this PR, so there were no codeowners initially and I've added it due to the audit criteria. @rbarkerSL Please give your opinion also.

**Comment by rbarkerSL** [comment]
I agree mirror node team should have full ownership of the repo; the intent is not to take over the ownership here.

I believe the default owner on this repo should be `@hashgraph/mirror-node` as opposed to `release-engineering-managers` and all of the devops-ci/devops-ci-committers lines can be changed over to the `@hashgraph/mirror-node` team as well.

@steven-sheehy do you agree?

**Comment by steven-sheehy** [comment]
Roger I would agree with that. But would also say I don't think RE should own the README as it's documentation that we frequently change and has no CI impact. Also, we frequently change the gitignore so would like to at least be added to that file.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Default code owners for entire repository
+*   @hashgraph/release-engineering-managers
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:2]:
```suggestion
*   @hashgraph/mirror-node
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# Protect the repository root files
+/README.md                                      @hashgraph/devops-ci @hashgraph/release-engineering-managers
+**/LICENSE                                      @hashgraph/release-engineering-managers
+
+# CodeCov configuration
+**/codecov.yml                                  @hashgraph/devops-ci @hashgraph/release-engineering-managers
+
+# Git Ignore definitions
+**/.gitignore                                   @hashgraph/devops-ci @hashgraph/release-engineering-managers
+**/.gitignore.*                                 @hashgraph/devops-ci @hashgraph/release-engineering-managers
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:30]:
```suggestion
/.github/                                       @hashgraph/devops-ci @hashgraph/mirror-node @hashgraph/release-engineering-managers
/.github/workflows/                             @hashgraph/mirror-node @hashgraph/devops-ci

# Codacy Tool Configurations
/config/                                        @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node
.remarkrc                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node

# Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
/CODEOWNERS                                     @hashgraph/release-engineering-managers

# Protect the repository root files
/README.md                                      @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node
**/LICENSE                                      @hashgraph/release-engineering-managers

# CodeCov configuration
**/codecov.yml                                  @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node

# Git Ignore definitions
**/.gitignore                                   @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node
**/.gitignore.*                                 @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/mirror-node
```

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

**Review by steven-sheehy — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**